client/plugin/scripts/popup.js

```diff
@@ -42,7 +42,7 @@ function displayResults(top_sites, top_context) {
     //, Matches: ' + value;
 
     var resultText = document.createElement('a');
-    var createAText = document.createTextNode(item);
+    var createAText = document.createTextNode(value.title);
```

Comment:
As I had commented earlier, we want it in this format  `Title | youtube.com`. You may also need to truncate title to like 40 chars or so. Please see the comment for details. Please update it to that, unless you have strong opinions. Please comment there as well.

Reply:
I am thinking page title truncated at 20 charas and base domain name for now, these changes need to be from backend side, will make the changes in this pr itself